### PR TITLE
Use an unstable prefix for MSC2885: Hidden read receipts  

### PR DESCRIPTION
**Pull request by SimonBrandner**
We should be using an unstable prefix as per [`a4e5b45` (#2285)](https://github.com/matrix-org/matrix-doc/pull/2285/commits/a4e5b45204b243b6b23226a0fcf5cf8a9b10cdc0)

### CODE REVIEW

**Review by t3chguy — APPROVED** [review]
Yup, thanks. Synapse doesn't support this anyway so not super critical :P

File: src/client.ts [hunk 1]
```diff
@@ -3650,7 +3650,7 @@ export class MatrixClient extends EventEmitter {
         }
 
         const addlContent = {
-            "m.hidden": Boolean(opts.hidden),
+            "org.matrix.msc2885.hidden": Boolean(opts.hidden),
```
Comment (t3chguy) [inline src/client.ts:3653]:
```suggestion
            "org.matrix.msc2285.hidden": Boolean(opts.hidden),
```

File: src/client.ts [hunk 2]
```diff
@@ -6429,7 +6429,7 @@ export class MatrixClient extends EventEmitter {
         const content = {
             "m.fully_read": rmEventId,
             "m.read": rrEventId,
-            "m.hidden": Boolean(opts ? opts.hidden : false),
+            "org.matrix.msc2885.hidden": Boolean(opts ? opts.hidden : false),
```
Comment (t3chguy) [inline src/client.ts:6432]:
```suggestion
            "org.matrix.msc2285.hidden": Boolean(opts ? opts.hidden : false),
```

**Review by t3chguy — CHANGES_REQUESTED** [review]
THAT COULD HAVE BEEN A FUN ONE TO DEBUG

**Comment by SimonBrandner** [comment]
@t3chguy, thanks for the massive save!

**Review by t3chguy — APPROVED** [review]

**Pull request merged**